erigon-lib/txpool/fetch.go

```diff
@@ -175,10 +174,10 @@ func (f *Fetch) receiveMessage(ctx context.Context, sentryClient sentry.SentryCl
 		}
 		return err
 	}
-
 	var req *sentry.InboundMessage
 	for req, err = stream.Recv(); ; req, err = stream.Recv() {
 		if err != nil {
+			f.logger.Error("[txpool.receiveMessage]", "err", err)
```

Comment:
users using our logs for understanding - is erigon working or not. 
Error log level - means node can't work without user's actions. 
This error is self-recoverable and can be at Debug level

Reply:
set to debug lvl